lib/sig_item_map.ml

```diff
@@ -0,0 +1,25 @@
+open Types
+
+type t =
+  value_description String_map.t
+  * module_declaration String_map.t
+  * (type_declaration * Ident.t) String_map.t
```

Comment:
You should use a record here instead of a tuple. The types being different for each of the members does not allow mistakes but a record will still make the code easier to read!